plugin.go

```diff
@@ -46,12 +47,13 @@ type Configurer interface {
 }
 
 type Plugin struct {
-	cfg         *Config
-	log         *zap.Logger
-	tracer      *sdktrace.TracerProvider
-	propagators propagation.TextMapPropagator
-	mdw         mdw
-	intcpt      intcpt
+	cfg            *Config
+	log            *zap.Logger
+	tracer         *sdktrace.TracerProvider
+	propagators    propagation.TextMapPropagator
+	mdw            mdw
+	intcpt         intcpt
+	temporalIntcpt temporalIntcpt
```

Comment:
I guess it's time to realize... we don't pay for the letters 😄 
`intcpt` -> `grpcInterceptor`
`temporalIntcpt` -> `temporalInterceptor`